ng/src/web/components/dialog/dialog.js

```diff
@@ -145,20 +218,26 @@ class Dialog extends React.Component {
           role="dialog"
           tabIndex="1"
           width={width}
+          height={height}
           posX={posX}
           posY={posY}
           innerRef={ref => this.dialog = ref}>
           {children({
             close: this.handleClose,
             getMoveProps: this.getMoveProps,
           })}
+          {resizable && <Resizer
+            onMouseDown={this.onMouseDownResize}
+          />}
         </DialogContainer>
       </DialogOverlay>
     );
   }
 };
 
 Dialog.propTypes = {
+  height: PropTypes.string,
```

Comment:
Should this really be only a string or can it be numberOrNumberString from proptypes.js?